tests/end_to_end/netsh_test.cpp

```diff
@@ -534,3 +534,53 @@ TEST_CASE("unpin program", "[netsh][programs]")
                   "======  ====  =====  =========  =============  ====================\n");
     REQUIRE(result == NO_ERROR);
 }
+
+TEST_CASE("xdp interface parameter", "[netsh][programs]")
+{
+    _test_helper_libbpf test_helper;
+
+    // Load a program pinned.
+    int result;
+
+    // Load program with pinpath and loopback interface name.
+    std::string output = run_netsh_command_va_arg(
+        handle_ebpf_add_program, &result, 4, L"droppacket.o", L"xdp", L"mypinpath", L"Loopback Pseudo-Interface 1");
+    REQUIRE(strcmp(output.c_str(), "Loaded with ID 196609\n") == 0);
+    REQUIRE(result == NO_ERROR);
+    output = _run_netsh_command(handle_ebpf_delete_program, L"196609", nullptr, nullptr, &result);
+    REQUIRE(result == NO_ERROR);
+    REQUIRE(output == "Unpinned 196609 from mypinpath\n");
+    REQUIRE(bpf_object__next(nullptr) == nullptr);
+
+    // Load program loopback interface index.
+    output = _run_netsh_command(handle_ebpf_add_program, L"droppacket.o", L"xdp", L"interface=1", &result);
```

Comment:
besides testing with interface=1, also add tests for "interface=loopback_0" and "interface=Loopback Pseudo-Interface 1".  All three work with "netsh int ip sh <string>" and should similarly work here.

Reply:
`"Loopback Pseudo-Interface 1"` tests is there already.